### DISCUSSION TITLE
Would you mind talking a little bit about rendercv and your own relevant experiences with it?

### DISCUSSION THREAD

**Discussion by Aithosa** [rendercv/rendercv#157] [General]
For me, I got the chance to know about rendercv by being recommended by someone else. And I am more interested in open-source software and the creative experience behind it and the developers' stories. And my own views and understanding of the open-source community are changing, so I'd be happy if you don't mind talking about your own experiences. This can be a casual, informal conversation at any time you feel comfortable.

Please feel free to let me know if you think it's okay.

**Comment by sinaatalay** [comment]
Hi, thank you for your interest. I would like to share my experience. I'm not sure what you have in mind. Do you want to create some kind of podcast, or do you want me to share my experience here?

**Reply by Aithosa** [reply]
A podcast is indeed a great idea, but it's a bit out of my league 😭. I've just felt curious about some things before and wanted to know about them, not really planning to do a very formal interview unless you want to do that. So far, I'm planning to communicate in text form.

Do you have any suggestions? Is it something like a normal dialogue format, or do you want to collect the questions and then do the answers in unison, or do you have any better notes?
